plugins/ern_v0.41.1+/react-native-image-picker_v2.0.0+/config.json

```diff
@@ -0,0 +1,58 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": [],
+    "permissions": [
+      "android.permission.CAMERA",
+      "android.permission.WRITE_EXTERNAL_STORAGE"
+    ],
+    "copy": [
+      {
+        "source": "android/src/main/res/*",
+        "dest": "lib/src/main/res/imagepicker"
+      }
+    ],
+    "replaceInFile": [
+      {
+        "path": "lib/src/main/java/com/imagepicker/utils/UI.java",
+        "string": "com.imagepicker.R",
+        "replaceWith": "com.walmartlabs.ern.container.R"
+      },
+      {
+        "path": "lib/src/main/java/com/imagepicker/ImagePickerModule.java",
+        "string": "R.style",
+        "replaceWith": "com.walmartlabs.ern.container.R.style"
+      },
+      {
+        "path": "lib/src/main/AndroidManifest.xml",
+        "string": "<application>",
+        "replaceWith": "<application>\n        <provider\n            android:name=\"com.imagepicker.FileProvider\"\n            android:authorities=\"${applicationId}.provider\"\n            android:exported=\"false\"\n            android:grantUriPermissions=\"true\">\n            <meta-data\n                android:name=\"android.support.FILE_PROVIDER_PATHS\"\n                android:resource=\"@xml/provider_paths\" />\n        </provider>"
```

Comment:
This here is pretty ugly, but I don't think we currently have a better way of injecting this provider element into application. @belemaire any suggestions?

Reply:
Agree on both points. This does not look good but we don't have a better way right now.
You did the right thing though by replacing the `<application>` tag, this way this won't destroy whatever the container had generated in this element (MiniApp activities) or any other plugin that also inject configuration inside *(even though I think this is the first plugin injecting into `<application>` afaik)*.

Reply:
Thanks for the reply! Yes, I think this is the first plugin that replaces `<application>` like that.. That's why I wanted to double check.